lua/neogit/config.lua

```diff
@@ -109,8 +109,11 @@ M.values = {
       ["c"] = "CommitPopup",
       ["L"] = "LogPopup",
       ["Z"] = "StashPopup",
+      ["A"] = "CherryPickPopup",
```

Comment:
It seems that these mappings are not set up in `neogit/lua/status.lua`

They are present on your master branch, just not in this branch

Reply:
Yea, the status buffer is in an upcoming branch. I didn't wanna co-mingle things in this PR, since it's already a lot.